sentry-core/src/main/java/io/sentry/core/SentryClient.java

```diff
@@ -38,7 +39,9 @@ public SentryClient(SentryOptions options, @Nullable Connection connection) {
 
     ITransport transport = options.getTransport();
     if (transport == null) {
-      transport = HttpTransportFactory.create(options);
+      // Canva interview change
+      // transport = HttpTransportFactory.create(options);
+      transport = new FakeHttpTransport(options);
```

Comment:
seems better to throw if there's no transport? 

is Canva/sentry-android just for jobs? If so, it's not 100% clear.... in the future somebody might not supply transport in the options and then wonder why nothing is working.

Reply:
Yeah good point, i'm going to keep this on a separate branch just for jobs